### PR TITLE
Opportunities advanced find members 1431

### PR DESCRIPTION
**Pull request by marcleblanc**
Updated the advanced form to have a drop down list of 'departments' when selecting the 'User department' field and also updated the 'Opt In' field to show a drop down list as well (used same options as Collab).

Updated the language files to have the proper key:value in them.

These changes are for fixing for the following bugs:

- When ‘Field’ “User department” is selected; the Value box does not offer the drop down list of departments used in profiles for consistent spelling. [Connex & Collab]
- When ‘Field’ “Opt In” is selected; the Value box does not offer the drop down list of mechanisms offered via the Opportunities Platform anymore – it is a free text box. [Connex; resolved on Collab]

[Opportunities Platform > Find Members > Advance Find #1431](https://github.com/gctools-outilsgc/gcconnex/issues/1431)

### CODE REVIEW

File: mod/missions/views/default/missions/element-select.php [hunk 1]
```diff
@@ -116,6 +116,22 @@
             ));
         	break;
             
+        case elgg_echo('missions:opt_in'):
+        	$content .= elgg_view('input/dropdown', array(
+                'name' => $dropdown_name . '_element',
+                'value' => '',
+                'options' => array(
```
Comment (markwooff) [inline mod/missions/views/default/missions/element-select.php]: These are the opt-in options available on GCcollab, there are more available for GCconnex that are missing.

**Comment by Phanoix** [comment]
travis is erroring out with : 
```
ReferenceError: Can't find variable: jQuery
at /home/travis/build/gctools-outilsgc/gcconnex/vendors/jquery/jquery-migrate-1.2.1.min.js:2
```

**Comment by lucbelliveau** [comment]
@Phanoix I notice jquery was updated to version 1.11.0, possibly during the installation process automatically?  @marcleblanc ?  Or was it intentional to fix something?

Either way, I notice in the CI this:

> WARN [watcher]: Pattern "/home/travis/build/gctools-outilsgc/gcconnex/vendors/jquery/jquery-1.11.0.min.js" does not match any file.

Not sure where this came from, but it should probably be updated to 1.11.1.min.js.

**Comment by Phanoix** [comment]
Seems to be hard-coded in:
* engine/lib/views.php
* js/tests/karma.conf.js
* views/installation/page/default.php

also came up in  vendors/jquery/jquery.min.map

**Comment by marcleblanc** [comment]
@lucbelliveau The jQuery version update to 1.11.1 was to fix the IE issue with the drop downs as per markwoof's note in the issue:

> Note: just tested Item 1 on GCcollab, can confirm the auto-suggest appears on Chrome - however it does not on IE. This is the bug that will need to be fixed.

**Pull request merged**